market/deal_ingest_snap.go

```diff
@@ -253,7 +258,7 @@ func (p *PieceIngesterSnap) AllocatePieceToSector(ctx context.Context, maddr add
 		vd.tmin = alloc.TermMin
 		vd.tmax = alloc.TermMax
 
-		maxExpiration = int64(piece.DealSchedule.EndEpoch + alloc.TermMax)
+		maxExpiration = int64(head.Height() + alloc.TermMax)
```

Comment:
made add 89(instead of 90) days in epoch as well to allow for a bigger pool?

Reply:
TermMax already contains that grace period.

f05 allocations set TermMin to deal duration and TermMax to deal duration+90 days